monitoring/buckets.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
nit: 2017 as some of this code is copied from an earlier file.

Reply:
Can I not get away with calling it a brand new remix? Done.